conf/app.routes

```diff
@@ -13,6 +13,8 @@ GET           /application/:id                                       uk.gov.hmrc
 POST          /application                                           uk.gov.hmrc.thirdpartyapplication.controllers.ApplicationController.create
 POST          /application/:id                                       uk.gov.hmrc.thirdpartyapplication.controllers.ApplicationController.update(id: java.util.UUID)
 
+PUT           /application/:id/cidrBlocks                            uk.gov.hmrc.thirdpartyapplication.controllers.ApplicationController.updateCidrBlocks(id: java.util.UUID)
```

Comment:
Would be tempted to call the endpoint something like `ipWhitelist`. That we're using CIDR blocks is an implementation detail.

Reply:
I think cidrBlocks represent better the data expected by the endpoint, as the client needs to send a collection of CIDR blocks. The implementation of the collection or how you store it doesn't matter to the client, but as a client you do need to send CIDR blocks (and not IP addresses and masks, for instance) as that is what the server expects, so I'm not sure it needs to be hidden the way an internal implementation detail would be.

I don't mind using ipWhitelist if you feel strongly about it though...

Reply:
I kind of do - we're describing the whitelist associated with the application. We could do that perfectly adequately without using CIDR blocks at all.

`/application/foo/cidrBlocks` feels a lot more ambiguous than `/application/foo/ipWhitelist`

Reply:
Sounds reasonable